pkg/k8s/kubernetes_exec.go

```diff
@@ -50,3 +52,31 @@ func Kubectl(parts ...string) error {
 	}
 	return nil
 }
+
+func CreateSecret(secret types.K8sSecret) error {
+	secretData := flattenSecretData(secret.KeyValues)
+
+	args := []string{"-n", secret.Namespace, "create", "secret", secret.Type, secret.Name}
+	args = append(args, secretData...)
+
+	res, secretErr := KubectlTask(args...)
+
+	if secretErr != nil {
+		return secretErr
+	}
+	if res.ExitCode != 0 {
+		fmt.Printf("[Warning] unable to create secret %s, may already exist: %s", "basic-auth", res.Stderr)
```

Comment:
Should this have `basic-auth"` hard-coded?

Reply:
oops :eyes: